packages/nc-gui/components/cell/Url.vue

```diff
@@ -92,7 +95,9 @@ watch(
       :ref="focus"
       v-model="vModel"
       :placeholder="isEditColumn ? $t('labels.enterDefaultUrlOptional') : ''"
-      class="outline-none text-sm w-full px-2 py-2 bg-transparent h-full"
+      class="outline-none focus:!outline-none focus:ring-0 !border-none !outline-none text-sm w-full px-2 py-2 bg-transparent h-full"
```

Comment:
I've left a comment on `!`. It seems the changes got overrode. Please double check.